iceberg/src/main/scala/com/nvidia/spark/rapids/iceberg/parquet/converter/FromIcebergShadedImplicits.scala

```diff
@@ -0,0 +1,318 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.iceberg.parquet.converter
+
+import java.util.Optional
+
+import org.apache.iceberg.shaded.org.apache.parquet.column.{Encoding => ShadedEncoding, EncodingStats => ShadedEncodingStats}
+import org.apache.iceberg.shaded.org.apache.parquet.column.statistics.{Statistics => ShadedStatistics}
+import org.apache.iceberg.shaded.org.apache.parquet.hadoop.metadata.{BlockMetaData => ShadedBlockMetaData, ColumnChunkMetaData => ShadedColumnChunkMetaData, ColumnChunkProperties => ShadedColumnChunkProperties, ColumnPath => ShadedColumnPath, CompressionCodecName => ShadedCompressionCodecName}
+import org.apache.iceberg.shaded.org.apache.parquet.internal.hadoop.metadata.{IndexReference => ShadedIndexReference}
+import org.apache.iceberg.shaded.org.apache.parquet.schema.{ColumnOrder => ShadedColumnOrder, GroupType => ShadedGroupType, LogicalTypeAnnotation => ShadedLogicalTypeAnnotation, MessageType => ShadedMessageType, PrimitiveType => ShadedPrimitiveType, Type => ShadedType}
+import org.apache.iceberg.shaded.org.apache.parquet.schema.LogicalTypeAnnotation.{LogicalTypeAnnotationVisitor => ShadedLogicalTypeAnnotationVisitor, TimeUnit => ShadedTimeUnit}
+import org.apache.iceberg.shaded.org.apache.parquet.schema.PrimitiveType.{PrimitiveTypeName => ShadedPrimitiveTypeName}
+import org.apache.parquet.column.{Encoding, EncodingStats}
+import org.apache.parquet.column.statistics.Statistics
+import org.apache.parquet.hadoop.metadata.{BlockMetaData, ColumnChunkMetaData, ColumnChunkProperties, ColumnPath, CompressionCodecName}
+import org.apache.parquet.internal.hadoop.metadata.IndexReference
+import org.apache.parquet.schema.{ColumnOrder, GroupType, LogicalTypeAnnotation, MessageType, PrimitiveType, Type, Types}
+import org.apache.parquet.schema.LogicalTypeAnnotation.TimeUnit
+import org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName
+import scala.collection.JavaConverters._
+
+
+
+object FromIcebergShadedImplicits {
+  implicit class ColumnOrderConverter(val columnOrder: ShadedColumnOrder) {
```

Comment:
Why use `implicit` key word? Seem it's unnecessary, right?
And also for the following `implicit`